ykrt/src/compile/jitc_yk/codegen/x86_64.rs

```diff
@@ -501,19 +511,45 @@ impl<'a> X64CodeGen<'a> {
         self.reg_into_new_local(inst_idx, WR0);
     }
 
+    fn codegen_arg(&mut self, inst_idx: InstrIdx, idx: u16) {
+        // For arguments passed into the trace function we simply inform the register allocator
+        // where they are stored and let the allocator take things from there.
+        match idx {
```

Comment:
Is this deciding which register to use for the ABI based on arg idx?

You can search `ARG_REGS` for this.

Reply:
Try [f6a4955](https://github.com/ykjit/yk/pull/1052/commits/f6a4955905df99338035c19b004174c02fd70758)